client/my-sites/plan-features/header.jsx

```diff
@@ -44,10 +45,7 @@ class PlanFeaturesHeader extends Component {
 		return (
 			<header className="plan-features__header" onClick={ this.props.onClick } >
 				{
-					popular &&
-						<div className="plan-features__header-banner">
-							{ translate( 'Our most popular plan' ) }
-						</div>
+					popular && <Ribbon className="plan-features__ribbon">{ translate( 'popular' ) }</Ribbon>
```

Comment:
The class name could be `plan-features__header-ribbon` so we continue the `__header-` naming convention we started here.